airbyte-integrations/connectors/source-scaffold-source-http/acceptance-test-docker.sh

```diff
@@ -1,16 +1,26 @@
 #!/usr/bin/env sh
 
+# Set this variable to false to bypass spec backward compatibility checks
+RUN_SPEC_BACKWARD_COMPATIBILITY_TEST=true
+
 # Build latest connector image
 docker build . -t $(cat acceptance-test-config.yml | grep "connector_image" | head -n 1 | cut -d: -f2-)
 
-# Pull latest acctest image
+# Pull latest source-acceptance-test image
 docker pull airbyte/source-acceptance-test:latest
 
+TEST_OPTIONS="--acceptance-test-config /test_input"
```

Comment:
we should probably(?) replace all `acceptance-test-docker.sh` files with this version. It shouldn't require any update of connectors. WDYT?

Reply:
not in this PR though as it would make it harder to read/revisit in the future

Reply:
I implemented Pedro's suggestion: enabling/disabling backward compatibility test run from `acceptance-test-config.yml`. Changes on the script become useless so I reverted all my modifications on `acceptance-test-docker.sh`.